Docker-Swarm-deployment/single-node/redis/redis-rejson-stack.yml

```diff
@@ -0,0 +1,42 @@
+version: '3.4'
+services:
+    redisrejson:
+      image: redisrejson
```

Comment:
tag the image appropriately, "redis-version-rejson-version" . This is important to track which version is being used.